data/upgrades/crew.json

```diff
@@ -878,6 +905,22 @@
             "type": "Coordinate",
             "difficulty": "White"
           }
+        ],
+        "grants": [
+          {
+            "action": {
+              "type": "Coordinate",
+              "difficulty": "White"
+            },
+            "value": 1
+          },
+          {
+            "action": {
+              "type": "Coordinate",
+              "difficulty": "Red"
+            },
+            "value": -1
```

Comment:
So a positive value would be to gain an action, and a negative value would be to lose an action?

Reply:
I wonder if the ship actually loses the red action if it gains a white action of the same type? Could it still choose to do the red action?

Reply:
I don't think we should remove the red action until we know if the new action replaces the old one or just gives you a new action. I think it's best to leave that for squad builders and such to take care of themselves for now. We can always add this later

Reply:
Under Actions in the rules reference (Page 3) it says "Some upgrade cards have an action bar that lists one or more actions. These actions are added to the ship's action bar and therefore are standard actions that the ship can perform".

This wording would imply that when you equip Tactical Officer it will add the White coordinate action. Nothing mentions removing or replacing thus giving you the option to self stress if you have an upgrade or reason to benefit from being stressed.